### PR TITLE
fixed api for gocd v21.2.0

### PR DESCRIPTION
**Pull request by Antonos**

### CODE REVIEW

File: client/components/TestReport.jsx [hunk 1]
```diff
@@ -59,8 +59,8 @@ const styles = {
 }
 
 // Chart.js setup
-Chart.defaults.global.defaultFontColor = white;
-Chart.defaults.global.defaultFontFamily = 'Roboto';
+// Chart.defaults.global.defaultFontColor = 'white';
+// Chart.defaults.global.defaultFontFamily = 'Roboto';
```
Comment (karmats) [inline client/components/TestReport.jsx:63]: Why was this removed?

File: server/services/GoService.js [hunk 2]
```diff
@@ -141,7 +141,7 @@ export default class GoService {
           let pipelineNameToGroupName = new Object();
           pipelineGroups.forEach((pipelineGroup) => {
             pipelineGroup.pipelines.map((pipeline) => {
-              if (pipelineNameToGroupName[pipeline.name] == undefined) {
+              if (pipelineNameToGroupName[pipeline.name] === undefined) {
```
Comment (karmats) [inline server/services/GoService.js:144]: ⭐

**Review by karmats — APPROVED** [review]

**Comment by karmats** [comment]
Thanks for this @Antonos and sorry for the late review. I'll merge this, but please answer my comment :)

File: client/components/TestReport.jsx (hunk 1 above)
Comment (Antonos) [reply client/components/TestReport.jsx:63]: I had an issue with fonts in the chrome dev tools console, UI was broken for me, once I commented out those lines - I was able to see pipelines, not sure if it doesn't work only for me or for others as well.

**Comment by karmats** [comment]
Looks like the [tests](https://github.com/karmats/gocd-monitor/runs/3697469664?check_suite_focus=true) fails. Can you look into that @Antonos ?

**Comment by Antonos** [comment]
@karmats 
https://github.com/karmats/gocd-monitor/pull/74
